openlibrary/core/vendors.py

```diff
@@ -2,6 +2,7 @@
 import web
 import urllib2
```

Comment:
This should be removed and the exception handling for the referenced `urllib2.HTTPError` reviewed and updated as necessary.